test/TankTests.cpp

```diff
@@ -105,6 +112,14 @@ TEST_F(TankTest, WhenTankIsOneAxisOutOfTheBoard_ThenShouldAllowToMoveOnlyOneAxis
   expect_vec2f_eq({0.f, 500.f}, mTankSUT.get_position());
 }
 
+TEST_F(TankTest, WhenTankTakesHitForZeroDamage_ThenShouldBeDestroyed) {
```

Comment:
I think a typo is in the test name, "ThenShouldNotBeDestroyed"

Reply:
Yes good catch :)